apis/infix-en_GB/atrium-api-infix-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/infix/en_GB/samples/deprecated/AnyAssertionSamples.kt

```diff
@@ -50,4 +61,188 @@ class AnyAssertionSamples {
             }
         }
     }
+
+    @Test
+    fun notToBe() {
+        expect(2) notToBe 3
+
+        fails {
+            expect(12) notToBe 12
+        }
+
+        expect(listOf(2)) notToBe listOf(3)
+
+        fails {
+            expect(listOf(2)) notToBe listOf(2)
+        }
+    }
+
+    @Test
+    fun isSameAs() {
+        val list = listOf(3)
+        expect(list) isSameAs list
+
+        fails {
+            expect(listOf(3)) isSameAs listOf(3) // fails because isSameAs is based on identity, use toBe for equality
+        }
+    }
+
+    @Test
+    fun isNotSameAs() {
+        expect(listOf(2)) isNotSameAs listOf(2) // holds because isSameAs is based on identity, use notToBe for equality
+
+        fails {
+            val list = listOf(3)
+            expect(list) isNotSameAs list // fails because isNotSameAs is based on identity, use toBe for equality
+        }
+    }
+
+    @Test
+    fun toBeNullIfNullGivenElse() {
+        expect<Int?>(null) toBeNullIfNullGivenElse null
+
+        expect<Int?>(1) toBeNullIfNullGivenElse { // subject inside this block is of type Int (actually 1)
+            it isLessThan 2
+        } // subject here is remains to type Int
```

Comment:
Not sure if this comment is necessary but if you want it, then you need to fix it
```suggestion
        } // subject here is back to type Int?
```